src/components/Header/Header.test.js

```diff
@@ -4,6 +4,10 @@ import renderer from 'react-test-renderer';
 import Header from './Header';
 import { withMemoryRouter } from 'utils';
 
+jest.mock('react-dom', () => ({
```

Comment:
I'll try to remember to add a comment as to why this has to be here.

Reply:
Added comment